Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -1303,6 +1307,39 @@ namespace AtomSampleViewer
         ReleaseRPIScene();
     }
 
+    void SampleComponentManager::EnableRenderPipeline(bool value)
```

Comment:
This name might be a little clearer
```SetRenderPipelineEnabled()```

Reply:
Thanks for the suggestion. Since it's not a variable to be set, but rather an action to do, to avoid possibly misleading to think it's a setter I incline a bit more towards `EnableRenderPipeline`.